codec-http2/src/test/java/io/netty/handler/codec/http2/Http2SecurityUtilTest.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2020 The Netty Project
+ *
+ * The Netty Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package io.netty.handler.codec.http2;
+
+import io.netty.buffer.UnpooledByteBufAllocator;
+import io.netty.handler.ssl.SslContext;
+import io.netty.handler.ssl.SslContextBuilder;
+import io.netty.handler.ssl.SslProvider;
+import io.netty.handler.ssl.SupportedCipherSuiteFilter;
+import org.junit.Assert;
+import org.junit.Assume;
+import org.junit.Test;
+
+import javax.net.ssl.SSLEngine;
+import javax.net.ssl.SSLException;
+
+public class Http2SecurityUtilTest {
+
+    @Test
+    public void testTLSv13CiphersIncluded() throws SSLException  {
+        Assume.assumeTrue(SslProvider.isTlsv13Supported(SslProvider.JDK));
+        SslContext content = SslContextBuilder.forClient().sslProvider(SslProvider.JDK).protocols("TLSv1.3")
+                .ciphers(Http2SecurityUtil.CIPHERS, SupportedCipherSuiteFilter.INSTANCE).build();
+        SSLEngine engine = content.newEngine(UnpooledByteBufAllocator.DEFAULT);
+        Assert.assertTrue(engine.getEnabledCipherSuites().length > 0);
```

Comment:
Not found of assertTrue without an explicit error message: failures are very hard to read.